docs/ReleaseNotes.md

```diff
@@ -1,289 +1,401 @@
+
+ - .asf
+ - .avi
+ - .dv
+ - .m4v
+ - .mov
+ - .mp4
+ - .mpg
+ - .mpeg
+ - .ogv
+ - .vp8
+ - .webm
+ - .wmv 
+
+You can also directly embed video URLs into your projects [using VineML](API/Vines.md#video).
+
+> **Note:** Videos currently will not play in the web editor, but they will play as expected on supported devices.
+
+
+![Better Video Support](/img/product/releases/2020.11/video_asset_support.gif)
+
+
+
+### Improvements
+
+#### My Experiences Screen
+
+In this update, we've revamped the entire My Experiences screen, making it faster than ever to find and create experiences! Not only can you create new projects and modify existing ones, but you can also search for them directly.
+
+We've also added user portraits on each experience card to tell you who has access to a file (you can check their names on mouseover). 
+
+Finally, we've also added the ability to sort your experiences by either Name or Recent Activity. No more are the days of spending minutes chasing down a file you just started working on!
+
+![My Experience Screen](/img/product/releases/2020.11/my_experiences_screen.gif)
+
+
+#### Drag & Drop Scripts
+
+We've added a bunch of new Public Scripts in 2020.11 that makes building interactions easier than ever! These Drag & Drop scripts allow even the least technical of folks to create complex simulations or immersive content. 
+
+To use one of these scripts, create a new asset and drag one of these  over to the Script section of the Inspector. You can then adjust different parameters, like the icon or color of a button, in the Variable window. Most importantly, you can also decide what message an interactable takes, and what message it sends on completion.  You could create a button that pulls apart a motor, or a proximity trigger that reveals a hidden message! 
+
+For more on Drag & Drop scripts, check out our official guide [here]().
+
+![Drag & Drop Scripts](/img/product/releases/2020.11/drag_and_drop_scripts.gif)
+
+
+#### Web Play Mode: Hand Interactions
+
+Building on its official debut in 2020.10, we've added a new feature to Web Play mode: hand interactions! You can now simulate grabbing particles (or fingerpainting) without needing to put on a HoloLens.
+
+![Hand Interactions](/img/product/releases/2020.11/fingerpainting.gif)
+
+
+#### Quality of Life Changes
+
+##### Script Tab: Script Names
+
+As of 2020.11, the names of a script now appears on the same line as the **File** button, right under the Script tab.
+
+![Script Names](/img/product/releases/2020.11/script_names.png)
+
+
+##### Web Editor User Portraits
+
+User portraits have also seen an overhaul in 2020.11. Collaborators on an experience can now be identified by the first letter of their first name, as well as by a randomly assigned color.
+
+![User Portraits](/img/product/releases/2020.11/user_portraits.png)
+
+
+### Fixes
+
+#### Misc. Bug Fixes
+
+- Fixed a bug that caused the web editor to crash when editing multiple scripts.

```

Comment:
This almost makes it sound like we support having 2 script editors open at the same time somehow. Maybe something more like "when entering and exiting the script editor a large number of times" (even though "large" here could only be like.... 5)